src/main/java/com/exadel/project/ProjectApplication.java

```diff
@@ -0,0 +1,19 @@
+package com.exadel.project;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@SpringBootApplication
+public class ProjectApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(ProjectApplication.class, args);
+    }
+
+    @GetMapping("/hello")
+    public String sayHello(@RequestParam(value = "myName", defaultValue = "World") String name) {
+        return String.format("Hello %s!", name);
+    }
```

Comment:
It would be better if `ProjectApplication` class has only one main method, so you can remove this method from here